pom.xml

```diff
@@ -381,6 +381,15 @@
           </executions>
         </plugin>
         <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-compiler-plugin</artifactId>
+          <configuration>
+            <!-- Forking temporarily enabled as it seems to workaround the wrong behavior of UF annotation processor, resulting
+                 in corrupted perspective definitions, etc. This needs to fixed directly in UF and the forking disabled again. -->
+            <fork>true</fork>
```

Comment:
The only way I see is to configure all the workbench modules to override the config, which is quite painful.

I am now looking into other alternatives (e.g. fixing the UF stuff or some advanced plugin config)